css/src/admin-global.scss

```diff
@@ -443,3 +443,7 @@ td.column-wpseo-linked {
 .copy-home-meta-description .yoast-help-panel {
 	max-width: 400px !important;
 }
+
+.ReactModal__Body--open {
```

Comment:
This is the default CSS class used by `react-modal`, it can be customized though: http://reactcommunity.org/react-modal/styles/classes.html#for-the-documentbody-and-html-tag